### ISSUE TITLE
cannot configure redis db number

### ISSUE THREAD

**Issue by njordr** [argaen/aiocache#93]
Hi.

I'm trying to set the db number for redis but it seems to be ignored:

```
aiocache.settings.set_defaults(
    class_='aiocache.RedisCache',
    endpoint='127.0.0.1',
    port=6379,
    namespace='test',
    db=2
)
await RedisCache().set(key='first_key', value='test value')
```

It always writes to db0

aiocache==0.2.1
python 3.5.2

**Comment by argaen** [comment]
You are right, same problem will happen with any other parameter different from `serializer`, `plugins` or `namespace` as you can see in https://github.com/argaen/aiocache/blob/master/aiocache/cache.py#L25.

I'll implement it for any other kwarg the backend classes can receive, it should be straightforward. Thanks for reporting!

--

Now, a side comment if you let me: The `set_defaults` & co utils are provided in case you are working without a framework that can help you to provide a single instance of the cache. In aiohttp for example you can set the cache instance in the beginning inside the app instance and then reuse that one, in sanic I believe https://github.com/channelcat/sanic/pull/244 (or something similar) will let you do this if it gets accepted. Why am I saying this? `set_defaults` works with global variables and its easy to mess with it. Although it provides the functionality and its useful, if you can rely in other mechanisms I encourage you to do so :).

**Comment by njordr** [comment]
Thanks a lot for your comment :D

**Comment by argaen** [comment]
Fixed by 15d6609b61af4493a4864d4b501a1c2a4bbfdad1
